packages/main/src/ComboBox.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui5-combobox-root ui5-input-focusable-element">
 	{{#if hasValueState}}
-		<span id="{{_id}}-valueStateDesc" class="ui5-hidden-text">{{ariaValueStateHiddenText}}</span>
+		<span id="value-state-description" class="ui5-hidden-text">{{ariaValueStateHiddenText}}</span>
```

Comment:
Why

Reply:
`this._id` is deprecated.
Shadow root is an isolated env, which means you can have same ID's in all instances.